pkg/climate/description.go

```diff
@@ -163,26 +163,14 @@ func (climate Climate) getDescription() (string, error) {
 
 	description += weather + "."
 
-	predatorCount := 0
```

Comment:
Since species doesn't have `EatsAnimals` or `AnimalType` I took this out and modified the conditionals below. Is this what you wanted?